modules/native_streaming_client_module/src/native_streaming_signal_impl.cpp

```diff
@@ -41,16 +41,22 @@ Bool NativeStreamingSignalImpl::onTriggerEvent(const EventPacketPtr& eventPacket
 void NativeStreamingSignalImpl::assignDomainSignal(const SignalPtr& domainSignal)
 {
     if (domainSignal.assigned())
-        if (domainSignal.asPtrOrNull<IMirroredSignalConfig>() == nullptr)
+    {
+        if (auto mirrorDomainSignal = domainSignal.asPtrOrNull<IMirroredSignalConfig>(); mirrorDomainSignal.assigned())
+        {
+            setMirroredDomainSignal(mirrorDomainSignal);  
+        }
+        else
         {
             throw NoInterfaceException(
                 fmt::format(R"(Domain signal "{}" does not implement IMirroredSignalConfig interface.)",
                             domainSignal.getGlobalId()));
         }
-    if (domainSignal.assigned())
-        setMirroredDomainSignal(domainSignal.asPtr<IMirroredSignalConfig>());
+    }
     else
+    {
```

Comment:
We've been moving towards omitting curly braces in 1-line if statements.

Reply:
I dont like when we have brackets on if. but dont have on else. btw in code style there is a note for this case